Scripts/Src/Program.cs

```diff
@@ -20,28 +31,79 @@ static int Main(string[] args)
             Console.WriteLine("Replacing " + bootstrapName + " to " + appName + " in " + bootstrapRootDir);
             string contents = "";
             var files = from string file in Directory.EnumerateFiles(bootstrapRootDir, "*", SearchOption.AllDirectories) where !file.Contains("bootstrap-script") select file;
+            string endOfLine = "";
+            int startIndex, endIndex;
             foreach (string file in files)
             {
                 contents = File.ReadAllText(file);
+                if(contents.Contains("\r\n")) endOfLine = "\r\n";
+                else if(contents.Contains("\n")) endOfLine = "\n";
                 if (contents.Contains(bootstrapName))
                 {
                     contents = contents.Replace(bootstrapName, appName);
                     if (file.Contains("/README.md"))
                     {
-                        contents = contents.Replace("### [Kickoff] Application Setup\n","")
-                                            .Replace("After cloning the bootstrap, run `chmod +x ./Scripts/script.sh`\n","")
-                                            .Replace("Then `./Scripts/script.sh AppName` where `AppName` is your application name.\n","")
-                                            .Replace("Your app is ready. Happy coding!\n","");
+                        contents = contents.Replace($@"### [Kickoff] Application Setup{endOfLine}","")
+                                            .Replace($@"After cloning the bootstrap, run `chmod +x ./Scripts/script.sh`{endOfLine}","")
+                                            .Replace($@"Then `./Scripts/script.sh AppName` where `AppName` is your application name.{endOfLine}","")
+                                            .Replace($@"If you don't need authentication run `./Scripts/script.sh AppName delete-auth`{endOfLine}","")
+                                            .Replace($@"Your app is ready. Happy coding!{endOfLine}","");
+                    }
+                    if(deleteAuth == DeleteAuthenticationParamValue())
+                    {
+                        if(file.Contains("/Startup.cs"))
+                        {
+                            Console.WriteLine("Updating Startup.cs ...");
+                            contents = contents.Replace($@"using Microsoft.AspNetCore.Identity.EntityFrameworkCore;{endOfLine}", "")
+                                                .Replace($@"using Microsoft.AspNetCore.Identity;{endOfLine}", "")
+                                                .Replace($@"app.UseIdentity();{endOfLine}            ", "");

```

Comment:
its ok this space?
.Replace($@"app.UseIdentity();{endOfLine}            ", "");